sdk/src/test/scala/com/horizen/api/http/SidechainBlockApiRouteTest.scala

```diff
@@ -223,116 +198,5 @@ class SidechainBlockApiRouteTest extends SidechainApiRouteTest {
         assertsOnSidechainErrorResponseSchema(entityAs[String], ErrorInvalidBlockHeight("", None).code)
       }
     }
-
-    "reply at /template" in {
```

Comment:
unfortunately, we need to add tests for newly added logic: start/stop forging, generateBlock.